static/js/settings_org.js

```diff
@@ -1197,6 +1197,22 @@ export function build_page() {
         update_message_edit_sub_settings(is_checked);
     });
 
+    $("#id_realm_delete_own_message_policy").on("change", (e) => {
+        const setting_value = Number.parseInt($(e.target).val(), 10);
+        const disable_limit_setting =
+            setting_value === settings_config.common_message_policy_values.by_admins_only.code;
+        settings_ui.disable_sub_setting_onchange(
+            !disable_limit_setting,
+            "id_realm_msg_delete_limit_setting",
+            true,
+        );
+        settings_ui.disable_sub_setting_onchange(
+            !disable_limit_setting,
+            "id_realm_message_content_delete_limit_minutes",
+            true,
+        );
+    });
```

Comment:
Could this share code with `set_delete_own_message_policy_dropdown`? I noticed that's the pattern for the similar `#id_realm_allow_message_editing`.